src/components/Hero.tsx

```diff
@@ -35,6 +35,10 @@ const Hero = () => {
 
   useKeys(["ControlLeft", "KeyK"], handleCmdK, { target: containerRef });
 
+  useKeys(["ControlRight", "KeyK"], handleCmdK, { target: containerRef });
```

Comment:
It wouldn't be clean to create a function to pass these parameters and apply DRY
````
function useKey(superKey, key, target, hasFocus) {
  useKeys(
    [superKey, key],
    async (e: KeyboardEvent) => {
      if (!hasFocus) {
      } else {

      }
      // prevent browser from handling CMD/CTRL + K
      e.preventDefault();
    },
    { target: target }
  );
}